src/Sylius/Bundle/AdminBundle/Resources/views/Product/Show/Content/PageBody/_details.html.twig

```diff
@@ -112,16 +114,16 @@
                                                     {% set currencyCode = channel.baseCurrency.code %}
                                                     {% set price = channelPricing.price %}
                                                     {% set originalPrice = channelPricing.originalPrice ?? 0 %}
-                                                    {% set lowestPriceBeforeDiscount = channelPricing.lowestPriceBeforeDiscount ?? 0 %}
+                                                    {% set lowestPriceBeforeDiscount = channelPricing.lowestPriceBeforeDiscount %}
 
-                                                    <tr>
-                                                        <td>
-                                                            <span class="badge me-2" style="background-color: {{ channel.color }};"></span>
+                                                    <tr {{ sylius_test_html_attribute('variants', "%s"|format(variant.name) ) }}>
```

Comment:
This will result in massive duplication as you're making a new variants=name attribute for each channel within the same variant